client_new/src/components/shared/Forms/LabeledTextArea.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+    /**
+     * An text input with a label on top
+     *
+     * One can use the label prop to display an _unstyled_ text on top of the textarea.
+     * In order to style the label text, a child element should be passed instead.
+     */
+
+    // The textarea value, must be set.
+    export let value: string
+
+    // An optional unstyled label text.
+    export let label: string = ''
+
+    // An optional prop passed to textarea.
+    export let maxlength: number = undefined
+</script>
+
+<label class="flex flex-col w-full">{label}<slot></slot>
```

Comment:
On pourrait ajouter une prop pour gérer l'accessibilité avec un `id` et un `for`. Qu'est-ce que tu en penses ?

Reply:
Vu ensemble, pas besoin.